rpcd/playbooks/roles/rpc_maas/templates/galera_check.yaml.j2

```diff
@@ -34,3 +34,65 @@ alarms      :
             if (percentage(metric["mysql_current_connections"], metric["mysql_max_configured_connections"]) > {{ mysql_connection_warning_threshold }} ) {
                 return new AlarmStatus(WARNING, "Mysql connectons has exceeded {{ mysql_connection_warning_threshold }}% of configured maximum");
             }
+    mysql_max_used_connections :
+        label                   : mysql_max_used_connections--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["mysql_max_seen_connections"] > metric["mysql_max_configured_connections"]) {
```

Comment:
I think this is similar to the previous one in that it isn't really possible for max_seen connections to be higher than max_connections, at best it can be the same.

What we probably want is more like the "percentage_used_mysql_connections" check which uses percentage - e.g. seen_connections as a percentage of max_connections, vs a threshold that is defined as a var (e.g. 95%).
That would probably be quite conservative since max_seen_connections never goes down until you flush status.